src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs

```diff
@@ -55,7 +55,7 @@ public RestoreSummary(bool success)
             ConfigFiles = configFiles.AsList().AsReadOnly();
             FeedsUsed = sourceRepositories
                 .Select(source => source.PackageSource.Source)
-                .ToList()
+                .AsList()
```

Comment:
This change is unnecessary. There's no functional change here. 
LINQ generates enumerates which had to be converted to List.